pace-util/pace/util/constants.py

```diff
@@ -31,24 +31,39 @@
 #####################
 # Physical constants
 #####################
-RADIUS = 6.3712e6  # Radius of the Earth [m] #6371.0e3
-PI = 3.1415926535897931  # 3.14159265358979323846
-OMEGA = 7.2921e-5  # Rotation of the earth  # 7.292e-5
-GRAV = 9.80665  # Acceleration due to gravity [m/s^2]
-RDGAS = 287.05  # Gas constant for dry air [J/kg/deg] # 287.04
-RVGAS = 461.50  # Gas constant for water vapor [J/kg/deg]
-HLV = 2.5e6  # Latent heat of evaporation [J/kg]
-HLF = 3.3358e5  # Latent heat of fusion [J/kg]  # 3.34e5
-RGRAV = 1.0 / GRAV  # Inverse of gravitational acceleration
-# CP_AIR: Specific heat capacity of dry air at
-# constant pressure [J/kg/deg] # RDGAS / KAPPA
-CP_AIR = 1004.6  # Heat capacity of dry air at constant pressure
-KAPPA = RDGAS / CP_AIR  # 2.0 / 7.0
+
+GFS_PHYS = False
+if GFS_PHYS:
```

Comment:
So.... which one do we actually need? The evil GFS_PHYS 😈

Reply:
Ported my comment from another branch, check it out

Reply:
```# The FV3GFS model ships with two sets of constants, one used in the GFS physics
# package and the other used for the Dycore. Their difference are small but significant
# Our Fortran executable on GCE has GFS_PHYS=True
GFS_PHYS = True
```